client/app/scripts/superdesk-users/views/edit-form.html

```diff
@@ -25,10 +25,7 @@
               <span sd-switch ng-model="$parent._active" ng-click="toggleStatus(_active)"></span>
             </div>
         </div>
-        <div class="error-holder">
-          <p class="alert alert-error" ng-show="error && !error.message" translate>Please fix the issues below.</p>
-          <p class="alert alert-error" ng-show="error && error.message" translate>{{error.message}}</p>
```

Comment:
Not always is the same error and the problem is that the notification disappear in 5s or so.
Probably in this case because is the same error we can hide one of them.
The line with Please fix issue below should not be deleted, it will be showed on input validation errors.
The second line can be deleted.